packages/data/src/redux-store/metadata/test/selectors.js

```diff
@@ -12,6 +12,20 @@ import {
 	hasFinishedResolution,
 	isResolving,
 } from '../selectors';
+/**
+ * WordPress dependencies
+ */
+import { createRegistry } from '@wordpress/data';
+
+jest.useFakeTimers();
```

Comment:
Would it be possible to write the same tests with `useRealTimers()`? And instead of calling `advanceTimersByTime`, do some `await`-ing where appropriate?